### PR TITLE
Task Update Docs

### PR DESCRIPTION
**Pull request by ianpersaud**

### CODE REVIEW

File: source/includes/tasks.md [hunk 1]
```diff
@@ -165,3 +165,126 @@ tags      | Includes a list of tags associated to the task
...
+
+
+
+
+
+## Update a Specific Task
+
+
+```http
+PATCH /api/v2/projects/1/tasks/1-T2/ HTTP/1.1
```
Comment (funkaoshi) [inline source/includes/tasks.md]: I thought you just send the 3 fields?

File: source/includes/tasks.md [hunk 2]
```diff
@@ -165,3 +165,126 @@ tags      | Includes a list of tags associated to the task
...
+    "title": "Secure forgotten password",
+    "updated": "2015-07-08T02:16:33.923315Z",
+    "url": "http://example.com/.../1-T2",
+    "verification_status": null
+}
+```
+
+Updates a single Task resource, as specified by the project and task id parameters.
+
+**`PATCH /api/v2/projects/{project_id}/tasks/{project_id}-{task_id}/`**
```
Comment (geoffwhittington) [inline source/includes/tasks.md]: `{project_id}-{task_id}` should be `{task_id}`. Going forward, our id fields are opaque - people should not assume any structure in them like they have with APIv1.

File: source/includes/tasks.md (hunk 1 above)
Comment (geoffwhittington) [inline source/includes/tasks.md]: I agree, we should only show at most the three supported fields in the payload

**Comment by funkaoshi** [comment]
@ianpersaud can you wrap this up.

**Comment by geoffwhittington** [comment]
@ianpersaud What is outstanding on this PR?

File: source/includes/tasks.md (hunk 2 above)
Comment (ianpersaud) [inline source/includes/tasks.md]: changed!

File: source/includes/tasks.md (hunk 1 above)
Comment (ianpersaud) [inline source/includes/tasks.md]: changed.

**Comment by funkaoshi** [comment]
+1

**Pull request merged**